datadog_checks_base/datadog_checks/base/utils/db/core.py

```diff
@@ -94,83 +99,83 @@ def compile_queries(self):
 
     def execute(self, extra_tags=None):
         """This method is what you call every check run."""
-        logger = self.check.log
```

Comment:
This needs to stay here b/c when we construct a QueryManager in a check's `__init__`  there is no check ID at that point https://github.com/DataDog/integrations-core/blob/47882add411a894f2782148343a43be88645c356/datadog_checks_base/datadog_checks/base/log.py#L47-L53